net/haproxy/src/opnsense/service/templates/OPNsense/HAProxy/haproxy.conf

```diff
@@ -526,30 +537,34 @@ defaults
 
 {% if helpers.exists('OPNsense.HAProxy.frontends') %}
 {%   for frontend in helpers.toList('OPNsense.HAProxy.frontends.frontend') %}
-{%     if frontend.enabled=='1' %}
+{%     if frontend.enabled == '1' %}
 # Frontend: {{frontend.name}} ({{frontend.description}})
 frontend {{frontend.name}}
-{#       # collect ssl certs (if configured) #}
-{%       if frontend.ssl_certificates|default("") != "" %}
-{%         set ssl_certs = [] %}
-{%         for cert in frontend.ssl_certificates.split(",") %}
-{%           do ssl_certs.append('crt /var/etc/haproxy/ssl/' ~ cert ~ '.pem') %}
-{%         endfor %}
-{%       endif %}
-{#       # advanced ssl options #}
-{%       if frontend.ssl_customOptions|default("") != "" %}
-{#         # add a space to separate it from other ssl params #}
-{%         set ssl_options = frontend.ssl_customOptions ~ ' ' %}
+{%       if frontend.ssl_enabled == '1' %}
+{#         # collect ssl certs (if configured) #}
+{%         if frontend.ssl_certificates|default("") != "" %}
+{%           set ssl_certs = [] %}
```

Comment:
...move this outside of the "if frontend.ssl_enabled" clause to reset it's value if SSL is not configured.